### PR TITLE
Add the orange juice sales v2 notebook

### PR DESCRIPTION
**Pull request by YunsongB**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

- Add the orange juice sales v2 notebook.

fixes #

### CODE REVIEW

File: sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb [hunk 1]
```diff
@@ -0,0 +1,1337 @@
...
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "from azure.core.exceptions import ResourceNotFoundError\n",
+        "from azure.ai.ml.entities import AmlCompute\n",
+        "\n",
+        "cluster_name = \"cpu-cluster\"\n",
```
Comment (vlbejan) [inline sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb:329]: Please change this to `oj-cluster`

File: sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb [hunk 2]
```diff
@@ -0,0 +1,1337 @@
...
+      "outputs": [],
+      "source": [
+        "forecast_horizon = n_test_periods"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "### Customize the Featurization Settings\n",
```
Comment (vlbejan) [inline sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb:437]: Please remove `the`.

File: sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb [hunk 3]
```diff
@@ -0,0 +1,1337 @@
...
+        "        ColumnTransformer(fields=[\"Price\"], parameters={\"strategy\": \"ffill\"}),\n",
+        "    ]\n",
+        "}"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "### Create the AutoML forecasting job."
```
Comment (vlbejan) [inline sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb:472]: Please remove `.` at the end.

File: sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb [hunk 4]
```diff
@@ -0,0 +1,1337 @@
...
+        "\n",
+        "# Training properties are optional\n",
+        "forecasting_job.set_training(blocked_training_algorithms=[\"ExtremeRandomTrees\"])"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## 4.2 Run the Command\n",
```
Comment (vlbejan) [inline sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb:536]: Please replace with `4.2 Train the AutoML model`.

File: sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb [hunk 5]
```diff
@@ -0,0 +1,1337 @@
...
+        "# Delete the batch endpoint and compute. Do not do it occasionally.\n",
+        "ml_client.batch_endpoints.begin_delete(name=batch_endpoint_name).wait()\n",
+        "ml_client.compute.begin_delete(name=cluster_name).wait()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## 6.3 Deployment\n",
```
Comment (vlbejan) [inline sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb:1075]: Please replace the entire Section 6.3 with the contents of [bike-share notebook](https://github.com/Azure/azureml-examples/blob/main/sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-bike-share/auto-ml-forecasting-bike-share.ipynb)

**Review by vlbejan — CHANGES_REQUESTED** [review]

File: sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb (hunk 1 above)
Comment (YunsongB) [reply sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb:329]: Changed.

File: sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb (hunk 2 above)
Comment (YunsongB) [reply sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb:437]: Changed.

File: sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb (hunk 3 above)
Comment (YunsongB) [reply sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb:472]: Changed.

File: sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb (hunk 4 above)
Comment (YunsongB) [reply sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb:536]: Done.

**Review by vlbejan — APPROVED** [review]

File: sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb [hunk 6]
```diff
@@ -0,0 +1,1318 @@
...
+        "Next, we upload the directory with the test set data which will be used in the batch end point inference."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "import os\n",
```
[the remark this answers is not in the document]
Comment (YunsongB) [reply sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb:294]: Removed.

File: sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb [hunk 7]
```diff
@@ -0,0 +1,1318 @@
...
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "output_file = \"forecast.json\"\n",
+        "batch_deployment = BatchDeployment(\n",
+        "    name=\"non-mlflow-deployment\",\n",
```
[the remark this answers is not in the document]
Comment (YunsongB) [reply sdk/python/jobs/automl-standalone-jobs/automl-forecasting-orange-juice-sales/automl-forecasting-orange-juice-sales-mlflow.ipynb:883]: Changed.

**Review by nick863 — APPROVED** [review]

**Review by nick863 — APPROVED** [review]

**Pull request merged**